### PR TITLE
make changes corresponding to RDW_Common

### PR DESCRIPTION
**Pull request by malaffoon**
The changes that go along with https://github.com/SmarterApp/RDW_Common/pull/133, as well as other changes to make it work with the latest RDW_Common.

### CODE REVIEW

File: admin-service/src/main/java/org/opentestsystem/rdw/admin/service/PipelineService.java [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+import org.springframework.security.access.prepost.PreAuthorize;
+
+import java.util.Collection;
+import java.util.List;
 import org.opentestsystem.rdw.admin.model.Pipeline;
 import org.opentestsystem.rdw.admin.model.PipelineScript;
 import org.opentestsystem.rdw.admin.model.PipelineTest;
 import org.opentestsystem.rdw.admin.model.PipelineTestRun;
+import org.opentestsystem.rdw.script.PipelineDefinition;
 import org.opentestsystem.rdw.script.ScriptError;
```
Comment (malaffoon) [inline admin-service/src/main/java/org/opentestsystem/rdw/admin/service/PipelineService.java]: @esotrope, i hope i got these changes right. Mostly it was renaming of classes so i think i'm good but please review critically. Or just be ready to fix things after i merge  ;)

File: aggregate-service/src/main/java/org/opentestsystem/rdw/olap/repository/impl/JdbcActiveAssessmentRepository.java [hunk 2]
```diff
@@ -7,10 +7,10 @@
 import org.springframework.stereotype.Repository;
 
 import java.util.List;
+import org.opentestsystem.rdw.multitenant.cache.TenantCacheable;
 import org.opentestsystem.rdw.olap.model.AggregateServiceCache;
 import org.opentestsystem.rdw.olap.repository.ActiveAssessmentRepository;
 import org.opentestsystem.rdw.reporting.common.model.ActiveAssessment;
-import org.opentestsystem.rdw.reporting.common.multitenant.TenantCacheable;
```
Comment (malaffoon) [inline aggregate-service/src/main/java/org/opentestsystem/rdw/olap/repository/impl/JdbcActiveAssessmentRepository.java]: There are a bunch of this change ... just switching to the TenantCacheable from RDW_Common.

File: build.gradle [hunk 3]
```diff
@@ -21,7 +21,7 @@ plugins {
 apply plugin: 'io.spring.dependency-management'
 
 // Set version dependencies. This allows the cli to force version, e.g. `-Pcommon=1.1.0-SNAPSHOT`
-String rdwCommonVersion = project.hasProperty("common") ? project.property("common") : "1.4.0-149"
+String rdwCommonVersion = project.hasProperty("common") ? project.property("common") : "1.4.0-SNAPSHOT"
```
Comment (malaffoon) [inline build.gradle]: will change once RDW_Common build is available

File: admin-service/src/main/java/org/opentestsystem/rdw/admin/service/impl/DefaultPipelineService.java [hunk 4]
```diff
@@ -52,14 +52,14 @@
             final PipelineRepository repository,
             final PipelineScriptRepository scriptRepository,
             final PipelineTestRepository testRepository,
-            final PublishedPipelineRepository publishedPipelineRepository,
+            final PublishedPipelineRepository PublishedPipelineRepository,
```
Comment (esotrope) [inline admin-service/src/main/java/org/opentestsystem/rdw/admin/service/impl/DefaultPipelineService.java]: don't think you meant to capitalize this

Comment (malaffoon) [reply admin-service/src/main/java/org/opentestsystem/rdw/admin/service/impl/DefaultPipelineService.java]: oops global s&r, good catch

**Pull request merged**